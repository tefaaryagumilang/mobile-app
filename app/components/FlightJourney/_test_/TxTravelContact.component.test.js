import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';
import TxTravelContact from '../TxTravelContact.component';

const spy = jest.fn();
const store = createStore(() => ({}));
const DecoratedForm = reduxForm({
  form: 'testForm',
  onSubmit: spy
}
)(TxTravelContact);

describe('TxTravelContact Component: PaymentSelection', () => {
  it('renders correctly', () => {
    const formValues = {};
    const accounts = [];
    const tree = renderer.create(
      <Provider store={store}>
        <DecoratedForm formValues={formValues} accounts={accounts} mockDate={true}/>
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
