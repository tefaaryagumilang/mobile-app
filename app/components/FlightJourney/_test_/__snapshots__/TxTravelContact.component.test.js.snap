// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TxTravelContact Component: PaymentSelection renders correctly 1`] = `
<View>
  <KeyboardAwareScrollView
    extraHeight={120}
    keyboardShouldPersistTaps="handled"
  >
    <View>
      <View
        style={
          Object {
            "backgroundColor": "#C10C19",
            "height": 5,
            "width": 375,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderBottomColor": "#ccc",
            "borderBottomWidth": 0.5,
            "paddingHorizontal": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#000000",
              "fontSize": 28,
              "paddingVertical": 20,
            }
          }
        >
          Detail Kontak
        </Text>
        <View>
          <View>
            <View>
              <Text
                numberOfLines={1}
                style={
                  Object {
                    "fontSize": 14,
                    "left": 0,
                    "position": "absolute",
                    "top": 35,
                  }
                }
              >
                First Name
              </Text>
            </View>
            <View
              onResponderRelease={[Function]}
              onStartShouldSetResponder={[Function]}
              pointerEvents="auto"
            >
              <View
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "borderBottomColor": "rgba(0, 0, 0, 0.38)",
                    "borderBottomWidth": 0.5,
                    "height": 64,
                    "paddingBottom": 8,
                    "paddingTop": 32,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "position": "absolute",
                      "top": 36,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.38)",
                        "fontSize": 16,
                      }
                    }
                  >
                    
                  </Text>
                </View>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                    }
                  }
                >
                  <TextInput
                    allowFontScaling={true}
                    autoCapitalize="none"
                    disableFullscreenUI={true}
                    editable={true}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onChangeText={[Function]}
                    onContentSizeChange={[Function]}
                    onFocus={[Function]}
                    placeholder=""
                    rejectResponderTermination={true}
                    selectionColor="#D72F33"
                    style={
                      Array [
                        Object {
                          "flex": 1,
                          "margin": 0,
                          "padding": 0,
                          "top": 2,
                        },
                        Object {
                          "color": "rgba(0, 0, 0, .87)",
                          "fontSize": 16,
                          "height": 24,
                        },
                        Object {
                          "color": "#000",
                          "fontSize": 24,
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "flexDirection": "row",
                    "height": 8,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgb(213, 0, 0)",
                          "fontSize": 0,
                          "opacity": 0,
                        }
                      }
                    />
                  </View>
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgba(0, 0, 0, .38)",
                          "fontSize": 12,
                          "opacity": 1,
                        }
                      }
                    />
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View>
            <View>
              <Text
                numberOfLines={1}
                style={
                  Object {
                    "fontSize": 14,
                    "left": 0,
                    "position": "absolute",
                    "top": 35,
                  }
                }
              >
                Last Name
              </Text>
            </View>
            <View
              onResponderRelease={[Function]}
              onStartShouldSetResponder={[Function]}
              pointerEvents="auto"
            >
              <View
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "borderBottomColor": "rgba(0, 0, 0, 0.38)",
                    "borderBottomWidth": 0.5,
                    "height": 64,
                    "paddingBottom": 8,
                    "paddingTop": 32,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "position": "absolute",
                      "top": 36,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.38)",
                        "fontSize": 16,
                      }
                    }
                  >
                    
                  </Text>
                </View>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                    }
                  }
                >
                  <TextInput
                    allowFontScaling={true}
                    autoCapitalize="none"
                    disableFullscreenUI={true}
                    editable={true}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onChangeText={[Function]}
                    onContentSizeChange={[Function]}
                    onFocus={[Function]}
                    placeholder=""
                    rejectResponderTermination={true}
                    selectionColor="#D72F33"
                    style={
                      Array [
                        Object {
                          "flex": 1,
                          "margin": 0,
                          "padding": 0,
                          "top": 2,
                        },
                        Object {
                          "color": "rgba(0, 0, 0, .87)",
                          "fontSize": 16,
                          "height": 24,
                        },
                        Object {
                          "color": "#000",
                          "fontSize": 24,
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "flexDirection": "row",
                    "height": 8,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgb(213, 0, 0)",
                          "fontSize": 0,
                          "opacity": 0,
                        }
                      }
                    />
                  </View>
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgba(0, 0, 0, .38)",
                          "fontSize": 12,
                          "opacity": 1,
                        }
                      }
                    />
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View>
            <View>
              <Text
                numberOfLines={1}
                style={
                  Object {
                    "fontSize": 14,
                    "left": 0,
                    "position": "absolute",
                    "top": 35,
                  }
                }
              >
                Phone
              </Text>
            </View>
            <View
              onResponderRelease={[Function]}
              onStartShouldSetResponder={[Function]}
              pointerEvents="auto"
            >
              <View
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "borderBottomColor": "rgba(0, 0, 0, 0.38)",
                    "borderBottomWidth": 0.5,
                    "height": 64,
                    "paddingBottom": 8,
                    "paddingTop": 32,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "position": "absolute",
                      "top": 36,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.38)",
                        "fontSize": 16,
                      }
                    }
                  >
                    
                  </Text>
                </View>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                    }
                  }
                >
                  <TextInput
                    allowFontScaling={true}
                    autoCapitalize="none"
                    disableFullscreenUI={true}
                    editable={true}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onChangeText={[Function]}
                    onContentSizeChange={[Function]}
                    onFocus={[Function]}
                    placeholder=""
                    rejectResponderTermination={true}
                    selectionColor="#D72F33"
                    style={
                      Array [
                        Object {
                          "flex": 1,
                          "margin": 0,
                          "padding": 0,
                          "top": 2,
                        },
                        Object {
                          "color": "rgba(0, 0, 0, .87)",
                          "fontSize": 16,
                          "height": 24,
                        },
                        Object {
                          "color": "#000",
                          "fontSize": 24,
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "flexDirection": "row",
                    "height": 8,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgb(213, 0, 0)",
                          "fontSize": 0,
                          "opacity": 0,
                        }
                      }
                    />
                  </View>
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgba(0, 0, 0, .38)",
                          "fontSize": 12,
                          "opacity": 1,
                        }
                      }
                    />
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View>
          <View>
            <View>
              <Text
                numberOfLines={1}
                style={
                  Object {
                    "fontSize": 14,
                    "left": 0,
                    "position": "absolute",
                    "top": 35,
                  }
                }
              >
                Email
              </Text>
            </View>
            <View
              onResponderRelease={[Function]}
              onStartShouldSetResponder={[Function]}
              pointerEvents="auto"
            >
              <View
                style={
                  Object {
                    "backgroundColor": "transparent",
                    "borderBottomColor": "rgba(0, 0, 0, 0.38)",
                    "borderBottomWidth": 0.5,
                    "height": 64,
                    "paddingBottom": 8,
                    "paddingTop": 32,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "position": "absolute",
                      "top": 36,
                    }
                  }
                >
                  <Text
                    numberOfLines={1}
                    style={
                      Object {
                        "color": "rgba(0, 0, 0, 0.38)",
                        "fontSize": 16,
                      }
                    }
                  >
                    
                  </Text>
                </View>
                <View
                  style={
                    Object {
                      "flexDirection": "row",
                    }
                  }
                >
                  <TextInput
                    allowFontScaling={true}
                    autoCapitalize="none"
                    disableFullscreenUI={true}
                    editable={true}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onChangeText={[Function]}
                    onContentSizeChange={[Function]}
                    onFocus={[Function]}
                    placeholder=""
                    rejectResponderTermination={true}
                    selectionColor="#D72F33"
                    style={
                      Array [
                        Object {
                          "flex": 1,
                          "margin": 0,
                          "padding": 0,
                          "top": 2,
                        },
                        Object {
                          "color": "rgba(0, 0, 0, .87)",
                          "fontSize": 16,
                          "height": 24,
                        },
                        Object {
                          "color": "#000",
                          "fontSize": 24,
                        },
                      ]
                    }
                    underlineColorAndroid="transparent"
                    value=""
                  />
                </View>
              </View>
              <View
                style={
                  Object {
                    "flexDirection": "row",
                    "height": 8,
                  }
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgb(213, 0, 0)",
                          "fontSize": 0,
                          "opacity": 0,
                        }
                      }
                    />
                  </View>
                  <View
                    style={
                      Object {
                        "bottom": 0,
                        "left": 0,
                        "paddingVertical": 4,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                      }
                    }
                  >
                    <Text
                      numberOfLines={1}
                      style={
                        Object {
                          "backgroundColor": "transparent",
                          "color": "rgba(0, 0, 0, .38)",
                          "fontSize": 12,
                          "opacity": 1,
                        }
                      }
                    />
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "paddingTop": 50,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "paddingRight": 20,
              }
            }
          >
            <View
              accessible={true}
              clickable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
            >
              <View
                accessible={true}
                clickable={true}
                isTVSelectable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={null}
              >
                <View
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "flexDirection": "row",
                        "marginBottom": 5,
                      },
                      Array [
                        Object {
                          "flexDirection": "row-reverse",
                          "justifyContent": "space-between",
                          "marginHorizontal": -20,
                          "paddingHorizontal": 20,
                        },
                        Object {},
                      ],
                    ]
                  }
                >
                  <Image
                    source="checkbox-unchecked.png"
                    style={
                      Array [
                        Object {
                          "height": 26,
                          "width": 26,
                        },
                        Object {
                          "height": 25,
                          "width": 25,
                        },
                      ]
                    }
                  />
                  <View />
                </View>
              </View>
            </View>
          </View>
          <Text
            style={
              Object {
                "color": "#000000",
              }
            }
          >
            Tambahkan Kontak ke Daftar Template
          </Text>
        </View>
        <Text
          style={
            Object {
              "color": "#ccc",
              "paddingVertical": 10,
            }
          }
        >
          Kita akan mengirimkan konfirmasi booking, pengembalian dana, atau pergantian jadwal
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#ECECEC",
          "paddingHorizontal": 50,
          "paddingVertical": 20,
        }
      }
    >
      <View
        accessible={true}
        clickable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "backgroundColor": "#FA3F70",
                "borderRadius": 30,
                "flexDirection": "row",
                "height": 47,
                "justifyContent": "center",
                "paddingHorizontal": 15,
              },
              undefined,
              Object {},
            ]
          }
        >
          <Text
            style={
              Object {
                "color": "#FFFFFF",
                "fontSize": 16,
              }
            }
          >
            LANJUTKAN
          </Text>
        </View>
      </View>
    </View>
  </KeyboardAwareScrollView>
</View>
`;
